<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QXShapingOptionsWidget</class>
 <widget class="QWidget" name="QXShapingOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Font Size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="fontSizeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>6</number>
        </property>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>80</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>120</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>140</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>160</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>250</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>300</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>350</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>400</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="bidiGroupBox">
     <property name="title">
      <string>Bidirection</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Direction</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="bidiDirectionComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>LTR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RTL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="bidiBreakOnLevelChangeCheckBox">
        <property name="text">
         <string>Break on Level change</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="bidiBreakOnScriptChangeCheckBox">
        <property name="text">
         <string>Break on script change</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="bidiResolveScriptsCheckBox">
        <property name="text">
         <string>Resolve scripts</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="copyTextButton">
       <property name="text">
        <string>Copy Decoded Text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="togglePanelPushButton">
       <property name="text">
        <string>Toggle Panel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
